frontend/packages/kubevirt-plugin/integration-tests/tests/models/importWizard.ts

```diff
@@ -46,60 +46,86 @@ export class ImportWizard extends Wizard {
   /**
    * Edits attributes of a NICs that are being imported from source VM.
    */
-  async updateImportedNICs() {
+
+  async editNic(nic: Network) {
+    const nicDialog = new NetworkInterfaceDialog();
+    await clickKebabAction(nic.name, KEBAP_ACTION.Edit);
+    await waitForNoLoaders();
+    const networks = await nicDialog.getNetworks();
+    if (networks.length > 0) {
+      await nicDialog.selectNetwork(networks[networks.length - 1]);
+      await nicDialog.fillName(nic.name);
+      const err = await checkForError(view.errorHelper);
+      if (err) {
+        return err;
+      }
+    } else {
+      throw Error('No available networks to assign imported NICs');
+    }
+    await click(view.confirmActionButton);
+    await waitForNoLoaders();
+    return null;
+  }
+
+  async getImportedNics() {
     const rows = await tableRows();
-    let importedNICs = rows.map((line) => {
+    return rows.map((line) => {
       const cols = line.split(/\t/);
       return {
         name: cols[networkTabCol.name],
       };
     });
+  }
+
+  async updateImportedNICs() {
+    let importedNICs = await this.getImportedNics();
     // TODO: This is horrible, but unfortunately no better way to dynamically extract only device names
     // without using ElementArrayFinder, which on the other hand may cause NoStaleElement Exceptions
     importedNICs = importedNICs.filter((_, i) => i % 3 === 0);
-
-    const NICDialog = new NetworkInterfaceDialog();
-    await asyncForEach(importedNICs, async (NIC) => {
-      await clickKebabAction(NIC.name, 'Edit');
-      await waitForNoLoaders();
-      const networks = await NICDialog.getNetworks();
-      if (networks.length > 0) {
-        await NICDialog.selectNetwork(networks[networks.length - 1]);
-      } else {
-        throw Error('No available networks to assign imported NICs');
-      }
-      await click(view.confirmActionButton);
-      await waitForNoLoaders();
+    await asyncForEach(importedNICs, async (nic) => {
+      await this.editNic(nic);
     });
   }
 
   /**
    * Edits attributes of Disks that are being imported from source VM.
    */
-  async updateImportedDisks() {
+  async editDisk(disk: Disk) {
+    const diskDialog = new DiskDialog();
+    await clickKebabAction(disk.name, 'Edit');
+    await waitForNoLoaders();
+    // This if is required to workaround bug 1814611.
+    // NFS is not supported for conversion PV and HPP should be used instead.
+    if (disk.name === 'v2v-conversion-temp' && STORAGE_CLASS === 'nfs') {
+      await diskDialog.selectStorageClass('hostpath-provisioner');
+    } else {
+      await diskDialog.selectStorageClass(STORAGE_CLASS);
+    }
+    await diskDialog.fillName(disk.name);
```

Comment:
why are we filling out the name? it should already have this value

Reply:
Removed it. That was leftover, cause in previous versions NIC name was incorrect and had to be replaced, now there is no need in it.